client/scss/grids/_grid_dividers.scss

```diff
@@ -1,21 +1,29 @@
 .grid--dividers {
   .grid__cell {
-    border-left: 1px solid color('accessible-grey-2');
-    padding-left: map-get($gutter-width, 'small')/2;
-    margin-left: map-get($gutter-width, 'small')/2 - 1;
+    position: relative;
 
-    @include respond-to('medium') {
-      padding-left: map-get($gutter-width, 'medium')/2;
-      margin-left: map-get($gutter-width, 'medium')/2 - 1;
-    }
+    &:before {
+      position: absolute;
+      content: '';
+      top: 0;
+      bottom: 0;
+      width: 1px;
+      background: color('keyline-grey');
```

Comment:
Hmm. Again, getting into nesting, BEM is great at keeping a flat structure, when you start using two paradigms, you lose the gain of it's simplicity.

Could we not have `grid-cell--divider` as it's a state on the cell, not the grid?